src/onboarding2/Configure/ConfigureTemplateScreens.js

```diff
@@ -60,7 +60,7 @@ function ConfigureTemplateScreens({
           <div
             css={`
               max-width: ${82 * GU}px;
-              padding: 0 ${3 * GU}px ${(below('medium') ? 9 : 0) * GU}px
+              padding: 0 ${3 * GU}px ${(below('medium') ? 9 : 3) * GU}px
```

Comment:
We use `below('medium')` and `layoutName === 'small'` so much, I wonder if we should have a `<CompactProvider>` + `useCompact` hook; it is quite verbose sometimes to fetch the info needed for one check, maybe one provider that has set the dependencies in place could help out.

Reply:
We could think of something like that yes, but I would avoid having hooks directly in the tree as it makes it easy to miss them or return earlier.

Also about the term “compact”, I don’t necessarily see it as an equivalent of small viewports, but more about the layout density of a given component. For example, Gmail has a “compact density” that can be used at any screen size.